### PR TITLE
Add JarInfer models for Android 11 (SDK 30)

### PR DESCRIPTION
**Pull request by lazaroclapp**
This is a new NullAway artifact to serve as platform models
for the `android.jar` classes for SDK level 30 / Android 11,
providing nullability information for Android platform classes
as determined by the JarInfer tool.

The binary model jarinfer.astubx produced from AOSP tag
`android-11.0.0_r34` by running JarInfer on it as
described in `RELEASING.md`.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -29,7 +29,7 @@ buildscript {
 plugins {
   id "com.github.sherter.google-java-format" version "0.8"
   id "net.ltgt.errorprone" version "0.7" apply false
-  id "com.github.johnrengelman.shadow" version "2.0.4" apply false
+  id "com.github.johnrengelman.shadow" version "6.1.0" apply false
```
Comment (msridhar) [inline build.gradle:32]: Did we mean to bump this dep in this diff?  Either way is ok with me

**Review by msridhar — APPROVED** [review]
LGTM

File: build.gradle (hunk 1 above)
Comment (lazaroclapp) [reply build.gradle:32]: Nope, was already planning to change this back. That will be on the other PR.

Comment (lazaroclapp) [reply build.gradle:32]: Actually, lets just land #461 first. Then this will auto-magically fix upon rebase, I think.

**Pull request merged**